entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -610,4 +614,11 @@ class EntityService(
 
         throw ResourceNotFoundException("Default instance of $expandedAttributeName not found in entity $entityId")
     }
+
+    internal fun compactedGeoPropertyKey(expandedPropertyKey: String): String =
+        NGSILD_GEO_PROPERTIES_PROPERTIES
+            .zip(NGSILD_GEO_PROPERTIES_TERMS)
+            .filter { it.first == expandedPropertyKey }
+            .map { it.second }
+            .first()
```

Comment:
it could go in `JsonLdUtils` in shared module as it will soon be needed by all the services.